analyzers/tests/SonarAnalyzer.UnitTest/CFG/Roslyn/ControlFlowBranchTest.cs

```diff
@@ -96,10 +107,12 @@ public void Method(bool condition)
              *            v
              *         Exit 4
              */
+            var entryBlock = cfg.EntryBlock;
             var initBlock = cfg.Blocks[1];
             var tryBlock = cfg.Blocks[2];
             var finallyBlock = cfg.Blocks[3];
-            var exitBlock = cfg.Blocks[4];
+            var exitBlock = cfg.ExitBlock;
+            entryBlock.Kind.Should().Be(BasicBlockKind.Entry);
```

Comment:
This should not be needed. `entryBlock` isn't used to assert branches below and we can remove it.
```suggestion
```